code/index.html

```diff
@@ -1,22 +1,205 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html id="top">
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="ut-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Space News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<body>
+  <!--Header with nav-bar, background and animation-->
+  <header class="header">
+    <!--navbar with links and searchbar, collapses on mobile devices-->
+    <nav class="nav">
+      <input type="checkbox" id="checkbox">
+      <label for="checkbox"><a class="menu-button"></a>
+      </label>
+      <div class="menu-content">
+        <a href="contacts.html" target="blank">contact</a>
+        <a href="about.html" target="blank">about</a>
+        <a href="signup.html" target="blank">sign up</a>
+        <a href="login.html" target="blank">login</a>
+      </div>
+      <div class="search-content">
+        <input type="checkbox" id="searchbox">
+        <div class="searchbar">
+          <input type="text" placeholder="Search..">
+        </div>
+        <label for="searchbox">
```

Comment:
We shouldn't use `label` tags as wrappers for images (or any different tag than `input`). They should only me implemented in two possible setups :

1. Label connected with input via id:
```
<input type="text" placeholder="Search..." id="search-input" />
<label for="search-input">Type phrase in searchbox</label>
```

1. Label connected with input via nesting:
```
<label>
    Type phrase in searchbox
    <input type="text" placeholder="Search..." />
</label>
```

Let's fix that!